javascript/javascript1/week3/homework.js

```diff
@@ -0,0 +1,207 @@
+//todo :first exercise
+//Item array removal
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+// find idx of nemeToRemove
+const idxOfNameToRemove = names.indexOf(nameToRemove);
+console.log(idxOfNameToRemove);
+
+// remove name
+console.log(names.splice(idxOfNameToRemove, 1));
+
+//original array
+console.log(names);
+
+//todo :second exercise
+//When will we be there??
+//* DATA
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+//* LOGIC
+const arrivalTime = function (speed, distance) {
+  const time = (distance / speed) * 60;
+  const minutes = Math.floor(time % 60);
+  const hours = Math.floor(time / 60);
+  let message1;
+  if (time >= 1) {
+    message1 = `${hours} hours and ${minutes} minutes`;
+  } else {
+    message1 = `${time} minutes`;
+  }
+  return message1;
+};
+//* rendering
+
+const travelTime = arrivalTime(
+  travelInformation.speed,
+  travelInformation.destinationDistance
+);
+console.log("--------secound exercise");
+console.log(travelTime); // 8 hours and 38 minutes
+
+//todo :third exercise
+//Series duration of my life
+
+//* DATA
+const seriesDurations = [
+  {
+    title: "Game of thrones",
+    days: 3,
+    hours: 1,
+    minutes: 0,
+  },
+  {
+    title: "Sopranos",
+    days: 3,
+    hours: 14,
+    minutes: 0,
+  },
+  {
+    title: "The Wire",
+    days: 2,
+    hours: 12,
+    minutes: 0,
+  },
+];
+
+//*LOGIC
+
+function lifeDurationHourly(age) {
+  let message = "";
+  let percentageTotal = 0;
+
+  // entire life hourly
+  const lifeByHours = age * 365 * 24;
+  for (const el of seriesDurations) {
+    // change movie duration by hour
+    const moveDurationByHour = el.days * 24 + el.hours;
+
+    // calculate percentage of life by hour
+    const percentageOfLife =
+      Math.trunc(
+        Number((moveDurationByHour / lifeByHours) * (100).toFixed(3)) * 1000
+      ) / 1000;
+
+    percentageTotal += percentageOfLife;
```

Comment:
I wonder if you've tried one thing and then have started building on top of it. This seems very overcomplicated.
Here is what I came up with:
```
    const percentageOfLife = ((moveDurationByHour / lifeByHours) * (100)).toFixed(3);

    percentageTotal += Number(percentageOfLife);
```